leapp/messaging/answerstore.py

```diff
@@ -115,9 +115,11 @@ def get(self, scope, fallback=None):
 
         :param scope: Scope of the data to retrieve.
         :param fallback: Fallback value to return if not found.
-        :return:
+        :return: A shallow copy of data stored in _storage by scope key
         """
-        answer = self._storage.get(scope, fallback)
+        # NOTE(ivasilev) self.storage.get() will return a DictProxy. To avoid TypeError during later
+        # JSON serialization a copy() should be invoked to get a shallow copy of data
+        answer = self._storage.get(scope, fallback).copy()
```

Comment:
@fernflower  you are right. Regarding the amount of existing and expected dialogs (and use case when it contains usually just one key) at all, it does not matter to create the shallow copy always.